templates/staff/project/list.html

```diff
@@ -15,12 +15,7 @@
 {% endblock breadcrumbs %}
 
 {% block hero %}
-{% url "staff:project-create" as staff_project_create_url %}
-{% #staff_hero title="Projects" %}
-  {% #button type="link" href=staff_project_create_url variant="primary" %}
-    Create an Interactive Project
-  {% /button %}
-{% /staff_hero %}
```

Comment:
Is this needed to close the `staff_hero` ?

Reply:
I thought that at first and had opening and closing template tags, but [it appears this is how slippers works](https://mitchel.me/slippers/docs/using-components/#block-and-inline-component-syntax).

Similar example in our code: https://github.com/opensafely-core/job-server/blob/3c25b965d6e6278333a3280d659cbee775b39eb4/templates/staff/redirect/list.html#L16-L18